cmd/preflight/cmd/check_container.go

```diff
@@ -198,7 +204,7 @@ func checkContainerRunE(cmd *cobra.Command, args []string, runpreflight runPrefl
 
 func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
-		return fmt.Errorf("a container image positional argument is required")
+		return fmt.Errorf("a container image positional argument is required, found %v", args)
```

Comment:
Is this updated needed for this task? It seems like an enhancement that might need it's own PR.

Reply:
i figured no harm in keeping it 🤷‍♂️